docs/README.md

```diff
@@ -1335,7 +1334,7 @@ You can use the GraphQL endpoint to query or subscribe to the read model records
 
 ## Features
 
-### IAM - Authentication and Authorization
+### Authentication and Authorization
```

Comment:
This might require a regeneration of the table of content

Reply:
done! b40504a